package.json

```diff
@@ -83,6 +83,7 @@
     "query-string": "^6.13.1",
     "rdf-ext": "^1.3.0",
     "react": "^16.13.1",
+    "react-bootstrap": "^1.3.0",
```

Comment:
We're intentionally not using react-bootstrap.

Reply:
Why not? This allows us to build less logic, which means we can focus more on the users problem rather than figuring out which tab should show.

Reply:
I spent a good day working on a solution with the jquery tab stuff provided by bootstrap, but React completely interrupts its functionality (by re-rendering) so eventually I replaced that with react-bootstrap that works around this. I guess we could make our own tabs components, is that what you want?

Reply:
I'd suggest taking a look at how we do resource "tabbing" (i.e., when more than one resource is open in the editor).